src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/utils/FunctionalTestBase.java

```diff
@@ -57,6 +58,27 @@ protected Response doPostRequest(final String path, final Map<String, String> ad
             .thenReturn();
     }
 
+    protected Response doPostRequestMultiPart(final String path, final Map<String, String> additionalHeaders,
+                                              String multiPartKey, final File multipartFile) {
+        return given()
+            .relaxedHTTPSValidation()
+            .headers(additionalHeaders)
+            .accept("*/*")
+            .multiPart(multiPartKey, multipartFile)
+            .when()
+            .post(path)
+            .thenReturn();
+    }
+
+    protected Response doDeleteRequest(final String path, final Map<String, String> additionalHeaders) {
```

Comment:
I can see that all the delete endoints in data-management does not have request body. You can keep this delete request but can you remove the one underneath as it is not needed.

Reply:
Removed the delete request without body